### PR TITLE
Fixed Profile Bug.

### PR DESCRIPTION
**Pull request by abs51295**
Resolves Issue #35.

Also, I updated the UI screen as suggested by @MeepyMay : 

profile-ui-update

### CODE REVIEW

File: app/src/main/java/com/systers/conference/profile/EditProfileActivity.java [hunk 1]
```diff
@@ -510,6 +514,9 @@ private void hideKeyboard() {
 
     private void saveChanges(boolean isBackPressed) {
         LogUtils.LOGE(LOG_TAG, mFirstName.getText().toString() + ' ' + AccountUtils.getFirstName(this));
+        final Intent intent = new Intent(this, MainActivity.class);
+        intent.putExtra("edit_profile", true);
```
Comment (chhavip) [inline app/src/main/java/com/systers/conference/profile/EditProfileActivity.java]: Put in strings.xml

Comment (abs51295) [inline app/src/main/java/com/systers/conference/profile/EditProfileActivity.java]: Shouldn't we create a public static constant? I think we discussed the same thing in previous PR. What do you think?

Comment (chhavip) [inline app/src/main/java/com/systers/conference/profile/EditProfileActivity.java]: Creating a constant uses up more space, strings.xml is a better option for now.

Comment (abs51295) [inline app/src/main/java/com/systers/conference/profile/EditProfileActivity.java]: @chhavip Done. Please review 😄

**Pull request merged**